plugins/scaffolder/src/translation.ts

```diff
@@ -298,5 +298,24 @@ export const scaffolderTranslationRef = createTranslationRef({
         editConfigurationTitle: 'Edit Configuration',
       },
     },
+    templateEditorToolbar: {
+      customFieldExplorerTooltip: 'Custom Fields Explorer',
+      installedActionsDocumentationTooltip: 'Installed Actions Documentation',
+      addToCatalogButton: 'Publish',
+      addToCatalogDialogTitle: 'Publish changes',
+      addToCatalogDialogContent: {
+        introduction:
+          'Follow the instructions below to create or update a template:',
+        step1: 'Save the template files in a local directory',
+        step2: 'Create a pull request to a new or existing git repository',
+        step3:
+          'If the template already exists, the changes will be reflected in the software catalog once the pull request gets merged',
+        step4:
+          'But if you are creating a new template, follow the documentation linked below to register the new template repository in software catalog',
+      },
+      addToCatalogDialogActions: {
+        closeButton: 'Close',
+      },
```

Comment:
Thinking this doesn't need the extra wrapping tbh, something like `addToCatalogDialogCloseButton`?

Reply:
It contains more inner keys now, please take a look.